packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -29,11 +51,12 @@ export function getCardConfig(card, sparkInstance) {
  * @returns {object}
  */
 
```

Comment:
It appears this may have been missed in your earlier PR, but there should not be any new lines between JSDoc and their target member.